apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -236,6 +238,34 @@ const Ticket = new GQLListSchema('Ticket', {
             knexOptions: { isNotNullable: true }, // Relationship only!
             kmigratorOptions: { null: false, on_delete: 'models.PROTECT' },
         },
+        propertyAddress: {
+            schemaDoc: 'Address of property, which synced with property and displayed, if property is deleted',
+            type: Text,
+            isRequired: true,
+            hooks: {
+                resolveInput: async ({ resolvedData, existingItem, fieldPath }) => {
```

Comment:
usually, you don't need to set these fields by hand. And resolve should not be called

Reply:
see comments above